.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Make sure to add to the .gitignore

Reply:
Added the following to .gitignore
.DS_store
package-lock.json
.env